vnext/src-win/Libraries/Text/Text.windows.js

```diff
@@ -144,25 +146,32 @@ const Text: React.AbstractComponent<TextProps, TextForwardRef> =
 
       let _selectable = selectable;
 
-      const processedStyle = flattenStyle(_style);
+      let processedStyle = flattenStyle<TextStyleProp>(_style);
```

Comment:
Why moving to let instead of const? It seems like the variable isn't reassigned.

Reply:
Good question, this is actually an upstream file: https://github.com/facebook/react-native/pull/45340/files, so this change comes from Meta!